### DISCUSSION TITLE
Survey Results and MVP

### DISCUSSION THREAD

**Discussion by sarahelsaig** [OrchardCMS/OrchardCore.Commerce#142] [General]
Hi everyone,

things are starting to pick up steam here. Thanks to your feedback we have a clearer impression of the community's priorities and some new developments are already coming in.

Let's see the topics that made it into the MVP as our short-term goals. Please take a look at the the open issues, we are looking for further insight, feedback and opinions. Especially about the topics of [tax infrastructure](https://github.com/OrchardCMS/OrchardCore.Commerce/issues/33) and [product discounts](https://github.com/OrchardCMS/OrchardCore.Commerce/issues/136).
- Basic inventory management
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/134
- Product variants
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/135
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/106
- Tax infrastructure
  - Check out the [Tax infrastructure](https://github.com/OrchardCMS/OrchardCore.Commerce/projects/2) project to track progress.
- Discounts
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/136
- Payment provider integration
  - [We've recently added Stripe which is what the vast majority of respondents wanted.](https://github.com/OrchardCMS/OrchardCore.Commerce/discussions/137)
- Assorted bugs
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/105
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/107
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/73
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/129
- Other priority issues
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/133
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/42
  - https://github.com/OrchardCMS/OrchardCore.Commerce/issues/128

(You can also track the progress [here](https://github.com/OrchardCMS/OrchardCore.Commerce/milestone/1).)

We won't forget about the other topics either, stage 2 after the MVP will contain them and some of your suggestions from the survey too. If you want to talk about your suggestion, please open a an issue and we can start planning together. Pull requests are also very welcome.

See you soon,
Sára

**Comment by Piedone** [comment]
It seems we're getting close to finishing the MVP! BTW I suggest changing [this link](https://github.com/OrchardCMS/OrchardCore.Commerce#:~:text=The%20work%20will%20focus%20at%20first%20on%20porting) to https://github.com/OrchardCMS/OrchardCore.Commerce/milestone/1 too (BTW that link in your message looks like it belongs to "Use existing CI workflows from Lombiq/GitHub-Actions instead of custom one (OCC-19) #128" only.)
